cmdstanpy_tutorial.py

```diff
@@ -29,32 +32,34 @@
 
 # ### Do Inference
 
-fit = model.optimize(data=data, show_console=True)
+fit = model.sample(data=data)
 print(fit)
 
 # ### Access the sample: the `CmdStanMCMC` object attributes and methods
 
-fit.draws().shape
-
-vars = fit.stan_variables()
-for (k, v) in vars.items():
-    print(k, v.shape)
+print(fit.draws().shape)
 
-thetas = fit.stan_variable(var='theta')
-pd.DataFrame(data=thetas).plot.density()
 
 # #### Get HMC sampler tuning parameters
 
-fit.step_size
-fit.metric_type
-fit.metric
+print(fit.step_size)
+print(fit.metric_type)
+print(fit.metric)
 
 # #### Summarize the results
 
-fit.summary()
+print(fit.summary())
 
 # #### Run sampler diagnostics
 
-fit.diagnose()
-
-
+print(fit.diagnose())
+
+# #### Visualize posterior density of parameter "theta"
+if sys.version[0:3] != '3.8':  # https://github.com/conda/conda/issues/9589
```

Comment:
I don’t understand the linked issue as a reason for this check? And I don’t think such a thing belongs in the tutorial code

Reply:
OK, will remove altogether.